src/content/docs/usage/api.mdoc

```diff
@@ -6,10 +6,10 @@ description: >-
   OpenAPI specifications. Access public instance or self-hosted API endpoints.
 ---
 
-You can find the api docs under `http://vikunja.tld/api/v1/docs` of your instance.<br  />
-A public instance is available on [try.vikunja.io](https://try.vikunja.io/api/v1/docs).
+You can find the api docs under `/api/v1/docs` of your instance. \
+A public instance is available on [https://try.vikunja.io/api/v1/docs](https://try.vikunja.io/api/v1/docs).
```

Comment:
Can't you just leave the url as is and it will be made clickable automatically?

Reply:
Doesn't seem so: I tried this first, but it wasn't converted to a link.

Reply:
Oh wow, maybe we need to enable that then. 

That's for another day, I'll merge this now.